### PR TITLE
micropython: updated to v1.9.2, micropython-lib: updated to v1.9

### PR DESCRIPTION
**Pull request by apatrushev**
Maintainer: Roger D (rogerdammit@gmail.com) / @roger- 
Compile tested: ar71xx, tl-wdr4300-v1, 17.01.2
Run tested: ar71xx, tl-wdr4300-v1, 17.01.2

Description:
Update micropython version to latest available v1.9.2, together with dependent library package.

Signed-off-by: Anton Patrushev <apatrushev@gmail.com>

### CODE REVIEW

**Comment by apatrushev** [comment]
I fixed Signed-off-by, but Travis still unhappy about it.

**Comment by hnyman** [comment]
> I fixed Signed-off-by, but Travis still unhappy about it.

You have NOT fixed the commit message itself.
Apparently you only edited the PR discussion comment that does not get stored into git.

**Comment by apatrushev** [comment]
Thank you for the direction.

File: lang/python/micropython/Makefile [hunk 1]
```diff
@@ -17,11 +17,11 @@ PKG_LICENSE_FILES:=LICENSE
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://github.com/micropython/micropython.git
-PKG_SOURCE_VERSION:=5a1d63fc14dae788f705403a43c2d8639b7dd9cd
+PKG_SOURCE_VERSION:=1f78e7a43130acfa4bedf16c1007a1b0f37c75c3
 
 PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
 PKG_SOURCE:=$(PKG_SOURCE_SUBDIR).tar.gz
```
Comment (diizzyy) [inline lang/python/micropython/Makefile]: Please change this to tar.xz and update PKG_MIRROR_HASH

File: lang/python/micropython-lib/Makefile [hunk 2]
```diff
@@ -17,11 +17,11 @@ PKG_LICENSE_FILES:=LICENSE
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://github.com/micropython/micropython-lib.git
-PKG_SOURCE_VERSION:=f81e979c56dddb771ad36ec381b7f2c6cd12111f
+PKG_SOURCE_VERSION:=f5fe55aaef1a39d3d56f07040c0ff9f7e841fdf7
 
 PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)-$(PKG_SOURCE_VERSION)
 PKG_SOURCE:=$(PKG_SOURCE_SUBDIR).tar.gz
```
Comment (diizzyy) [inline lang/python/micropython-lib/Makefile]: Please change this to tar.xz and update PKG_MIRROR_HASH

**Comment by champtar** [comment]
ping @roger-

**Comment by roger-** [comment]
Looks good to me.

**Pull request merged**